src/run/run.c

```diff
@@ -649,6 +649,29 @@ static int transient_cgroup_set_properties(sd_bus_message *m) {
                 if (r < 0)
                         return log_error_errno(r, "Failed to mangle name '%s': %m", arg_slice);
 
+                if (!endswith(arg_slice, ".slice") && slice[0] == '-') {
```

Comment:
maybe introduce an explicit syntax for this? i.e. if arg_slice starts with "+", then replace it with our own slice?

Reply:
or maybe even have --slice-inherit= in addition to --slice? that sounds less surprising?

Reply:
ack